symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/config/BdkCoreConfig.java

```diff
@@ -44,8 +47,14 @@ public ApiClient datafeedAgentApiClient(ApiClientFactory apiClientFactory) {
   }
 
   @Bean(name = "podApiClient")
-  public ApiClient podApiClient(ApiClientFactory apiClientFactory) {
-    return apiClientFactory.getPodClient();
+  public ApiClient podApiClient(ApiClientFactory apiClientFactory, @Nullable AuthSession authSession, BdkConfig config) {
+    ApiClient client = apiClientFactory.getPodClient();
+    if (config.getCommonJwt().getEnabled()) {
+      final OAuthSession oAuthSession = new OAuthSession(authSession);
```

Comment:
Is it working when the `authSession` is `null`?

Reply:
And `AuthSession` could be `Optional` (an example [here](https://github.com/finos/symphony-bdk-java/pull/620/files#diff-4fd58766e7a40ef56f93bcc4ce0405d5ead73f8edadb2b1bbfd15221a79b8052R28))

Reply:
Indeed if we are in oboMode only authSession is null and this was causing NPE in the OAuthSession. I added a check to only add authentication when authSession is not null. 
Also added a test to cover this use case. https://github.com/finos/symphony-bdk-java/pull/618/commits/038daede4e4e37dd03eb617f26c70aaa8af53c18

Reply:
I'm double checking if I have to do the same for Bdk Core as well